.github/workflows/balena-ci.yml

```
@@ -5,19 +5,22 @@ on:
       - master
 
 jobs:
-  build:
+  balena-cloud-build:
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: true
       matrix:
-        fail-fast: true
         fleets: [testbot-rig, testbot-rig-beta, testbot-personal]
     steps:
       - uses: actions/checkout@v2
-      - uses: balena-io/balena-ci@v0.3.2
+      - uses: balena-io/deploy-to-balena-action@master
+        id: build
         with:
           balena_token: ${{ secrets.BALENA_TOKEN }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
           fleet: balena/${{ matrix.fleets }}
```

```suggestion
          fleet: ${{ matrix.fleets }}
```
See related comment above
> Use fleet slugs to simplify adding new fleets from other orgs if needed.